ui/src/shared/constants/tableGraph.js

```diff
@@ -8,6 +8,9 @@ export const TIME_FORMAT_CUSTOM = 'Custom'
 
 export const TIME_COLUMN_DEFAULT = {internalName: 'time', displayName: ''}
 
+export const DEFAULT_SORT = 'asc'
+export const ALT_SORT = 'desc'
```

Comment:
these constants could be `ASCENDING` and `DESCENDING` and then DEFAULT_SORT could be one or the other

Reply:
ok. 'asc' and 'desc' is what lodash requires. but yes it's nice to make it readable